features/inventory/viewing_product_variant_with_specify_quantity_of_items_on_hand.feature

```diff
@@ -0,0 +1,43 @@
+@inventory
+Feature: Seeing product's variant with specify quantity of items on hand
+    In order to see how many variants of selected product are available on hand
+    As an Administrator
+    I want to be able to see quantity of variants available on hand for a selected product
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a product "T-shirt banana" priced at "€12.54"
+        And the product "T-shirt banana" has "Yellow" variant priced at "€20.54"
+        And the product "T-shirt banana" has "Green" variant priced at "€5.54"
+        And there are 5 items of product "T-shirt banana" in variant "Yellow" available in the inventory
+        And there are 5 items of product "T-shirt banana" in variant "Green" available in the inventory
+        And the store has a product "Skirt Sheep" priced at "€34.54"
+        And the product "Skirt Sheep" has "Red" variant priced at "€5.54"
```

Comment:
Product with a single variant is a simple product. When talking about product with a single variant we only talk about product and don't mention the variant at all. `the store has a product "X"`, `there are 5 units of product "X"  available in the inventory` etc.